### ISSUE TITLE
Failed to read the configuration file

### ISSUE THREAD

**Issue by sirdrakeistda** [nextcloud/spreed#5115]
I'm running the arm64 Version on a Raspberry Pi (the Raspberry Pi runs a 64bit system too). The Matterbridge stopped working after the Nextcloud Update to 20.0.5. It is still not working with NC 20.0.7. 

```
level=info msg="Running version 1.22.0 afcd362c" prefix=main
level=fatal msg="Failed to read configuration file: &os.PathError{Op:"open", Path:"/tmp/bridge-aqjb26fz.toml", Err:0x2}" prefix=config
```

**Comment by Lumrenion** [comment]
Are you using docker on you Raspi? I had the same issue whenever I removed the docker container and started it again because removing the container removes all directories that are not shared to the host by a volume. 

The issue was addressed here #4776 and should be fixed with the next release (see #5040) - which hopefully comes soon. Until then, clicking on *save* in the talk channel settings should be sufficient to fix the issue, as it saves the `.toml` file.
